compiler/src/iree/compiler/Codegen/LLVMCPU/test/BUILD.bazel

```diff
@@ -41,6 +41,7 @@ iree_lit_test_suite(
             "pipeline_peel_and_vectorize_tests.mlir",
             "pipeline_split_reduction_tests.mlir",
             "pipeline_tests.mlir",
+            "pipeline_tests_ssve.mlir",
```

Comment:
nit: `pipeline_ssve_tests.mlir`. All the pipeline files ends with `_tests.mlir`.